geode-dunit/src/main/java/org/apache/geode/internal/cache/functions/TestFunction.java

```diff
@@ -1041,6 +1044,20 @@ private void executeGetNetworkHop(FunctionContext context) {
     context.getResultSender().lastResult(networkHopType);
   }
 
+  private void executeSlowFunction(FunctionContext context) {
+    int entries = 4;
+    int waitBetweenEntriesMs = 5000;
+    for (int i = 0; i < entries; i++) {
+      try {
+        Thread.sleep(waitBetweenEntriesMs);
+      } catch (InterruptedException e) {
+        e.printStackTrace();
```

Comment:
I see other methods in this class are calling printStackTrace, but I'm not sure thats the best behavior here.